gui/packages/ubuntupro/lib/pages/landscape/landscape_page.dart

```diff
@@ -113,6 +179,56 @@ class LandscapePage extends StatelessWidget {
   }
 }
 
+class LandscapeErrorDialog extends StatelessWidget {
+  const LandscapeErrorDialog({
+    super.key,
+    required this.title,
+    required this.content,
+    required this.details,
+    required this.actions,
+  });
+
+  final String title;
+  final String content;
+  final String details;
+  final List<ButtonStyleButton> actions;
+
+  @override
+  Widget build(BuildContext context) {
+    final lang = AppLocalizations.of(context);
+    return AlertDialog(
+      title: Text(title),
+      content: SizedBox(
+        width: 640,
+        child: Column(
+            mainAxisSize: MainAxisSize.min,
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(content),
+              const SizedBox(height: 8),
+              YaruExpandable(
+                expandButtonPosition: YaruExpandableButtonPosition.start,
+                header: Text(lang.landscapeDetails),
+                child: Row(
+                  children: [
+                    Expanded(
+                      child: SelectableText(details),
+                    ),
+                  ],
+                ),
+              ),
+              const SizedBox(height: 16),
+              Text(
+                lang.landscapeProceedAnywayHint,
+                style: Theme.of(context).textTheme.bodySmall,
```

Comment:
Also, in my opinion, this should be the same size as the other text since generally I think that's what has been recommended in the past for other areas.